### PR TITLE
Switching button from bottom to top of letter

### PR DESCRIPTION
**Pull request by iros**

### CODE REVIEW

File: src/components/Letter.scss [hunk 1]
```diff
@@ -25,7 +25,9 @@
     background: #D9D9D9;
     color: #616161;
     display: block;
-
+    margin: 1em auto 0;
+    margin-top: 0px;
+    margin-bottom: 1em;
```
Comment (kadamwhite) [inline src/components/Letter.scss]: could be condensed to `margin: 0 auto 1em`

**Review by kadamwhite — CHANGES_REQUESTED** [review]

**Review by kadamwhite — APPROVED** [review]
@iros please review

**Comment by kadamwhite** [comment]
button order

**Comment by kadamwhite** [comment]
disregard background coloring issues in gif, those are recording artifacts: use it as a reference for button positioning

**Comment by iros** [comment]
Looks good.

**Pull request merged**